packages/react/collapsible/src/Collapsible.tsx

```diff
@@ -56,13 +57,19 @@ const Collapsible = React.forwardRef<CollapsibleElement, CollapsibleProps>(
       onChange: onOpenChange,
     });
 
+    const isAnimationDisabledRef = React.useRef(open);
```

Comment:
nit: Would a naming adjustment better communicate the temporary nature? something like `isAnimationPreventedRef` ? for some reason I associate `disabled` with props / semi permanent state.